config/routes.rb

```diff
@@ -10,6 +10,7 @@
       delete :sign_out, to: "sessions#logout"
       get :signed_in, to: "sessions#logged_in"
       
+      get '/api/v1/reservations', to: 'reservations#index'
```

Comment:
That route is already defined at: `resources :users, :cities, :reservations, :courses` making this line unnecessary. You may delete it.